src/screens/Dashboard/ToDoList.js

```diff
@@ -71,37 +75,73 @@ const ToDoList = ({route, navigation}) => {
         setSelectedDate(newDay.toLocaleDateString('en-US'));
     }
     useEffect(() => {
+        setLoading(true);
+        dispatch(getAllTasks()).then(resp =>{
```

Comment:
why do you re-fetch the taskData from the backend each time the user goes to the ToDoList page? Wouldn't it make more sense to fetch it once after Login?

Reply:
Yeap, this is correct position. I forgot about that possibility